apps/web/src/hooks/useTemplates.ts

```diff
@@ -12,7 +12,11 @@ export type INotificationTemplateExtended = INotificationTemplate & {
 };
 
 /** allow override of paginated inputs */
-export function useTemplates(pageIndex?: number, pageSize?: number) {
+export function useTemplates({
+  pageIndex,
+  pageSize,
+  areSearchParamsEnabled = false,
+}: { pageIndex?: number; pageSize?: number; areSearchParamsEnabled?: boolean } = {}) {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** Please derive these params from the `UsePaginationOptions` instead ‏